mobile/native/src/db/models.rs

```diff
@@ -672,6 +672,8 @@ pub(crate) struct PaymentInsertable {
     pub updated_at: i64,
     #[diesel(sql_type = Text)]
     pub description: String,
+    #[diesel(sql_type = Nullable<Text>)]
+    pub invoice: Option<String>,
```

Comment:
🤔 Do we really need the full invoice here? The payment already contains most of its properties anyways.

Reply:
We are displaying the raw invoice, so I would say yes. Maybe this means we can remove the description column though for instance.

Reply:
but is the raw invoice really useful? why not simply display the contents of the invoice which is already mostly present on the payment record.

Reply:
> but is the raw invoice really useful? why not simply display the contents of the invoice which is already mostly present on the payment record.

I simply implemented it because it's there in the Figma - maybe the user wants to inspect for their own details or for debugging purposes. In any case, IMO its helpful for our own debugging to have the real invoice around. This is the authoritative truth of what the payment is.

Ultimately, maybe @bonomat can clarify the reasoning here as he made the Figma mock?

Reply:
> Ultimately, maybe @bonomat can clarify the reasoning here as he made the Figma mock?

Mhmm.. better show and and don't need it than not show it and need it? 🤷‍♂️

I've modeled it after screens I've seen on other lightning apps.